app/views/kaminari/_paginator.html.erb

```diff
@@ -0,0 +1,23 @@
+<%# The container tag
+  - available local variables
+    current_page:  a page object for the currently displayed page
+    total_pages:   total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+    paginator:     the paginator that renders the pagination tags inside
+-%>
+<%= paginator.render do -%>
+  <nav class="pagination" role="navigation" aria-label="pager">
```

Comment:
Are the first/last page links missing here, or were the views just added for completeness in case we want to use them in the future?

Reply:
Yeah, these views were generated by Kaminari's initializer. I didn't modify them except for removing first/last as they weren't in @sigmaben's design